unit_tests/IssueUnitTests.d

```diff
@@ -0,0 +1,108 @@
+/**
```

Comment:
Perhaps this should be but in a specific sub directory and name the file `Issue46`.